src/components/templates/FireBarrel/useVideo.tsx

```diff
@@ -2,34 +2,6 @@ import { useCallback, useEffect, useState } from "react";
 import Video, { LocalVideoTrack } from "twilio-video";
 import firebase from "firebase/app";
 
-// export const VideoContext = createContext(null);
```

Comment:
I think it is wise not to have commented code like this in our codebase as it's unclear why it was commented.

Reply:
Agreed; this code has been unused for a long time. 💥 and away it goes! Great catch!